src/ui/public/paginated_table/__tests__/index.js

```diff
@@ -47,13 +47,19 @@ describe('paginated table', function () {
     };
   };
 
-  const renderTable = function (cols, rows, perPage, sort) {
+  const renderTable = function (cols, rows, perPage, sort, showBlankRows) {
     $scope.cols = cols || [];
     $scope.rows = rows || [];
     $scope.perPage = perPage || defaultPerPage;
     $scope.sort = sort || {};
+    $scope.showBlankRows = showBlankRows;
 
-    $el = $compile('<paginated-table columns="cols" rows="rows" per-page="perPage" sort="sort">')($scope);
+    const template = '<paginated-table columns="cols"'
```

Comment:
What do you think about using template literals to create the template markup instead of concatination?
```
    const template = `
      <paginated-table
        columns="cols"
        rows="rows"
        per-page="perPage"
        sort="sort"
        show-blank-rows="showBlankRows">`;
```